### DISCUSSION TITLE
compile and set config path

### DISCUSSION THREAD

**Discussion by mmattel** [ImageMagick/ImageMagick#3901] [General] [answered]
This may be a stupid question, but I have not found an answer for...

When compiling, I can set the `--prefix=` option which defaults for many paths.

If I want to just define the configuration path having the others default. I have not found a way doing it or I did not understand it.

`magick identify -list configure`
`CONFIGURE_PATH        /usr/local/etc/ImageMagick-7/` 

I would like to set the path to `/etc/ImageMagick-7/` without `/usr/local`
(same as it was with `/etc/ImageMagick-6/`)

**Comment by snibgo** [comment, marked as answer]
I don't fully understand the build process, so I am guessing: "./configure --help" and the source code of the configure scripts suggest the configure option "--sysconfdir=DIR".

**Reply by mmattel** [reply]
I did a test and can tell that this works great, see: https://github.com/SoftCreatR/imei/issues/32#issuecomment-874605762

**Reply by mmattel** [reply]
Note, it would be great if the configure options would be explained a bit better...
